functions/helpers.bash

```diff
@@ -331,17 +331,10 @@ is_raspios() {
   [[ "$(cat /boot/issue.txt)" =~ "Raspberry Pi reference" ]]
   return $?
 }
-# Debian/Raspbian oldoldoldstable
-is_stretch() {
-  if [[ "$osrelease" == "stretch" ]]; then return 0; fi
-  [[ $(cat /etc/*release*) =~ "stretch" ]]
-  return $?
-}
-# Debian/Raspbian oldoldstable
-is_buster() {
-  if [[ "$osrelease" == "buster" ]]; then return 0; fi
-  [[ $(cat /etc/*release*) =~ "buster" ]]
-  return $?
+# Represents the current OS versions we officially support
+is_supported() {
+  if is_bullseye || is_bookworm || is_sid || is_jellyfish || is_noble; then return 0; fi
```

Comment:
```suggestion
  if is_bullseye || is_bookworm || is_jellyfish || is_noble; then return 0; fi
```

Reply:
sid is the never-changing name of the ever-unstable release of Debian so I don't think we should support it here